content/packages.md

```diff
@@ -26,7 +26,7 @@ The `stable` and `current` release channels support the following package reposi
 - APT (Debian and Ubuntu platforms)
 - Yum (Enterprise Linux platforms)
 
-You can download Chef Software's GPG public key from [packages.chef.io](https://packages.chef.io/chef.asc).
+You can download Chef Software's GPG public key from [chefdownload-commercial.chef.io](https://chefdownload-commercial.chef.io/chef.asc).
```

Comment:
I don't think you can do this procedure with this API. That's a question for @sonarora